tests/spread/spread_01_2d.composite_bspline_43.mpirun=4.output

```diff
@@ -0,0 +1,35 @@
+Number of elements: 1280
+
+IBFEMethod: mesh part 0 is using FIRST order LAGRANGE finite elements.
+
+IBHierarchyIntegrator::initializePatchHierarchy(): tag_buffer = 3
+INSStaggeredHierarchyIntegrator::initializeCompositeHierarchyData():
+  projecting the interpolated velocity field
+rank: 0 level: 1 boxes:
+[(0,0),(3,3)]
+[(0,4),(3,7)]
+[(4,0),(7,3)]
+[(4,4),(7,7)]
+rank: 1 level: 1 boxes:
+[(0,8),(3,11)]
+[(0,12),(3,15)]
+[(4,8),(7,11)]
+[(4,12),(7,15)]
+rank: 2 level: 1 boxes:
+[(8,0),(11,3)]
+[(8,4),(11,7)]
+[(12,0),(15,3)]
+[(12,4),(15,7)]
+rank: 3 level: 1 boxes:
+[(8,8),(11,11)]
+[(8,12),(11,15)]
+[(12,8),(15,11)]
+[(12,12),(15,15)]
+
+rank: 0
```

Comment:
I think N needs to be bigger to get some values printed here.

Reply:
Hrumm.

Reply:
Damn Fortran. I lopped off the last argument of the composite B-spline 4/3 spreading subroutine.

Reply:
Take another look please. 😁